### PR TITLE
Add comments for the Java analyzer

### PR DESCRIPTION
**Pull request by sanderploegsma**
See https://github.com/exercism/java-analyzer/pull/85.

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]
Nice

**Comment by sanderploegsma** [comment]
@ErikSchierboom thanks for reviewing! I don't have write access so I can't merge it myself. Could you hit the merge button?

**Pull request merged**
